plugins/azure-devops/src/routes.ts

```diff
@@ -16,6 +16,11 @@
 
 import { createRouteRef } from '@backstage/core-plugin-api';
 
+export const azurePullRequestDashboardRouteRef = createRouteRef({
```

Comment:
Should this be plural - `azurePullRequestsDashboardRouteRef`?

Reply:
I don't think so. It's like this on ADO:
![image](https://user-images.githubusercontent.com/55280588/143583653-eb30e363-f5df-4840-9771-2f2343d3b523.png)

Reply:
Works for me, the screen shot for this Pull Request has "Azure PRs" in the side bar and we called the page `PullRequestsPage` so that's where my suggestion came from